src/prognostic_equations/advection.jl

```diff
@@ -286,6 +286,10 @@ NVTX.@annotate function explicit_vertical_advection_tendency!(Yₜ, Y, p, t)
         ᶜq_tot = @. lazy(specific(Y.c.ρq_tot, Y.c.ρ))
         vtt = vertical_transport(ᶜρ, ᶠu³, ᶜq_tot, FT(dt), energy_q_tot_upwinding)
         vtt_central = vertical_transport(ᶜρ, ᶠu³, ᶜq_tot, FT(dt), Val(:none))
+        if energy_q_tot_upwinding == Val(:first_order_kid)
+            vtt = vertical_transport_kid(ᶜρ, ᶠu³, ᶜq_tot, FT(dt), FT(t))
+            vtt_central = NullBroadcasted()  # turn off implicit advection
+        end
         @. Yₜ.c.ρq_tot += vtt - vtt_central
```

Comment:
```suggestion
        vtt_bc = ᶜρq_tot_vertical_transport_bc(ᶠu³, p.atmos.prescribed_flow, t)
        @. Yₜ.c.ρq_tot += vtt - vtt_central + vtt_bc
```
If the implicit tendency change is reverted, you can simplify this explicit term by keeping the current first-order upwinding tendency (minus the implicit sound wave component) and just adding a new boundary term, which could look something like
```julia
    function ᶜρq_tot_vertical_transport_bc(ᶠu³, flow::PrescribedFlow{FT}, t) where {FT}
        ρ_sfc = FT(1.1650101)
        q_tot_sfc = FT(0.014778325)
        u₃_sfc = Geometry.WVector(flow(FT(0), t))
        ᶜadvdivᵥ = Operators.DivergenceF2C(;
            bottom = Operators.SetValue(ρ_sfc * q_tot_sfc * u₃_sfc),
        )
        return @. lazy(-(ᶜadvdivᵥ(zero(ᶠu³)))
    end
```
Also, since this term is treated explicitly, it should probably be defined here in `advection.jl` instead of `implicit_tendency.jl`.